support-frontend/assets/pages/digital-subscription-landing/components/hero/hero.jsx

```diff
@@ -59,7 +59,7 @@ function CampaignHeader({ promotionCopy, countryGroupId }: PropTypes) {
   const title = promotionCopy.title || <>Subscribe for stories<br />
     <span css={yellowHeading}>that must be told</span></>;
 
-  const promoCopy = promotionHTML(promotionCopy.description, { css: paragraph, tag: 'p' });
+  const promoCopy = promotionHTML(promotionCopy.description, { css: paragraph, tag: 'div' });
```

Comment:
is the css still "paragraph" if it's a "div" now?  I guess this was just lining up DS with Paper and Weekly though?

Reply:
This is only in the context of the promo code; the styles are applied to a `<p>` tag normally.